# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/libwebrtcglue/moz.build
# Commit: 3fb02a012faa
# Full Hash: 3fb02a012faadb59c6a4ca42bfb50bcbb1037d93
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 5
# Description:
#   Bug 1654112 - Route packets with MediaEvents, and clean up conduits' cross-thread syncing. r=bwc,ng
#   
#   This patch solves some issues with the SSRC switching that can be triggered by
#   both signaling and incoming packets, on different threads. But mainly, because
#   that setup is inherently complex, this patch greatly simplifies the SSRC
# ==============================================================================

diff -r 336f58a54bd8 -r 3fb02a012faa dom/media/webrtc/libwebrtcglue/moz.build
--- a/dom/media/webrtc/libwebrtcglue/moz.build	Wed Aug 25 20:28:12 2021 +0200
+++ b/dom/media/webrtc/libwebrtcglue/moz.build	Thu Sep 09 17:42:30 2021 +0200
@@ -12,6 +12,7 @@
     "/ipc/chromium/src",
     "/media/libyuv/libyuv/include",
     "/media/webrtc",
+    "/third_party/libsrtp/src/include",
     "/third_party/libwebrtc",
     "/third_party/libwebrtc/third_party/abseil-cpp",
 ]